webrtc.html

```diff
@@ -2385,82 +2386,95 @@
+          <dt>readonly attribute DOMString? sdpMid</dt>
 
           <dd>If present, this contains the identifier of the "media stream
           identification" as defined in [[!RFC5888]] for the media component this
           candidate is associated with.</dd>
 
-          <dt>unsigned short sdpMLineIndex</dt>
+          <dt>readonly attribute unsigned short? sdpMLineIndex</dt>
 
-          <dd>This indicates the index (starting at zero) of the <a>media description</a> in the
-          SDP this candidate is associated with.</dd>
+          <dd>If present, this indicates the index (starting at zero) of the
+          <a>media description</a> in the SDP this candidate is associated with.</dd>
 
-          <dt>DOMString foundation</dt>
+          <dt>readonly attribute DOMString foundation</dt>
 
           <dd>A unique identifier that allows ICE to correlate candidates that
           appear on multiple <code><a>RTCIceTransport</a></code>s.</dd>
 
-          <dt>unsigned long priority</dt>
+          <dt>readonly attribute unsigned long priority</dt>
 
-          <dd>The assigned priority of the candidate. This is automatically
-          populated by the browser.</dd>
+          <dd>The assigned priority of the candidate.</dd>
 
-          <dt>DOMString ip</dt>
+          <dt>readonly attribute DOMString ip</dt>
 
           <dd>The IP address of the candidate.</dd>
 
-          <dt>RTCIceProtocol protocol</dt>
+          <dt>readonly attribute RTCIceProtocol protocol</dt>
 
           <dd>The protocol of the candidate (<code>udp</code>/<code>tcp</code>).</dd>
 
-          <dt>unsigned short port</dt>
+          <dt>readonly attribute unsigned short port</dt>
 
           <dd>The port of the candidate.</dd>
 
-          <dt>RTCIceCandidateType type</dt>
+          <dt>readonly attribute RTCIceCandidateType type</dt>
 
-          <dd>The type of candidate.</dd>
+          <dd>The type of the candidate.</dd>
 
-          <dt>RTCIceTcpCandidateType tcpType</dt>
+          <dt>readonly attribute RTCIceTcpCandidateType? tcpType</dt>
 
           <dd>If <code>protocol</code> is <code>tcp</code>, <code>tcpType</code> represents
-          the type of TCP candidate. Otherwise, <code>tcpType</code> is not present in the
-          dictionary.</dd>
+          the type of TCP candidate. Otherwise, <code>tcpType</code> is <code>null</code>.</dd>
 
-          <dt>DOMString relatedAddress</dt>
+          <dt>readonly attribute DOMString? relatedAddress</dt>
 
           <dd>For a candidate that is derived from another, such as a relay or reflexive
           candidate, the <dfn>relatedAddress</dfn> is the IP address of the candidate that
           it is derived from. For host candidates, the <code>relatedAddress</code>
-          is not present in the dictionary.</dd>
+          is <code>null</code>.</dd>
 
-          <dt>unsigned short relatedPort</dt>
+          <dt>readonly attribute unsigned short? relatedPort</dt>
 
```

Comment:
In my opinion it would be best to stick with the three old ones, since that's what current applications are expecting, and anything more would be redundant information.